configure.ac

```diff
@@ -92,6 +92,13 @@ if test "$enable_polkit" != "no"; then
 fi
 if test "$enable_ssh" != "no"; then
   PKG_CHECK_MODULES(libssh, [libssh >= 0.8.5])
+  if test "$pkg_failed" != "yes"; then
```

Comment:
Can this even happen? PKG_CHECK_MODULES's default action-if-not-found is to fail configure. We don't use this anywhere else, and it looks rather hackish?

Reply:
Good point, I'll remove the conditional.